### ISSUE TITLE
WP group 

### ISSUE THREAD

**Issue by gargul03** [abp-filters/abp-filters-anti-cv#469]
Ads are back: https://github.com/abp-filters/abp-filters-anti-cv/issues/183 + https://github.com/abp-filters/abp-filters-anti-cv/issues/380 + https://github.com/abp-filters/abp-filters-anti-cv/issues/381

**Comment by krystian3w** [comment]
Pages have `anti-adblock` so can't by reparired in ABP by "Eyeo" policy?

**Comment by adblock-filters** [comment]
This filter might help a bit: #471

**Comment by wizmak** [comment]
@krystian3w 
> Pages have `anti-adblock` so can't by reparired in ABP by "Eyeo" policy?

Not really, as long as we don't intentionally target anti-adblock messages/walls .

**Comment by wizmak** [comment]
> This filter might help a bit: #471

@gargul03 as @adblock-filters mentioned, #471 should help on some subdomains, please keep in mind this is usually a non-stop game.
